doc/attr.xml

```diff
@@ -1068,6 +1067,95 @@ gap> ArticulationPoints(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="StrongOrientation">
+<ManSection>
+  <Oper Name="StrongOrientation" Arg="D"/>
+  <Attr Name="StrongOrientationAttr" Arg="D"/>
+  <Returns>A digraph or <K>fail</K>.</Returns>
+  <Description>
+    A <E>strong orientation</E> of a connected symmetric digraph <A>D</A> (if
+    it exists) is a strongly connected subdigraph <C>C</C> of <A>D</A> such
+    that for every edge <C>[u, v]</C> of <A>D</A> either <C>[u, v]</C> or
+    <C>[v, u]</C> is an edge of <C>C</C> but not both.  By Robbin's Theorem,
+    only bridgeless digraphs (see <Ref Prop="IsBridgelessDigraph"/>) admit a
+    strong orientation. 
+    <P/>
+
+    This operation returns a strong orientation of the digraph <A>D</A> if 
+    <A>D</A> is symmetric and <A>D</A> admits a strong orientation. If <A>D</A>
+    is symmetric but does not admit a strong orientation, then <K>fail</K> is
+    returned. If <A>D</A> is not symmetric, then an error is given. 
+    <P/>
+
+    The method used in this operation has complexity <M>O(m+n)</M> where
+    <M>m</M> is the number of edges and <M>n</M> is the number of vertices in
+    the digraph.
+<Example><![CDATA[
+gap> StrongOrientation(DigraphSymmetricClosure(CycleDigraph(5))) 
+> = CycleDigraph(5);
+true
+gap> D := DigraphSymmetricClosure(Digraph(
+> [[2, 7], [3, 5], [4], [2], [6], [1], []]));;
+gap> IsBridgelessDigraph(D);
+false
+gap> StrongOrientation(D);
+fail
+gap> StrongOrientation(NullDigraph(0));
+<immutable empty digraph with 0 vertices>
+gap> StrongOrientation(DigraphDisjointUnion(CompleteDigraph(3), 
+>                                           CompleteDigraph(3)));
+fail
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="Bridges">
+<ManSection>
+  <Oper Name="Bridges" Arg="D"/>
+  <Attr Name="BridgesAttr" Arg="D"/>
```

Comment:
```suggestion
  <Attr Name="Bridges" Arg="D"/>
```

Reply:
So, there are other instances in this file where both the oper and attr version are documented, so doing this seems to be consistent (even if maybe not desirable)

Reply:
I'm fine with that where oper and attr versions exist (as with `StrongOrientation`) - i.e. for 'attributes' that return digraphs. But `Bridges` doesn't have an oper and an attr version...

Reply:
Doh, right you are...

Reply:
Fixed